package.json

```diff
@@ -17,28 +17,31 @@
         "url": "https://github.com/aws-observability/aws-rum-web.git"
     },
     "scripts": {
-        "server": "npm run build:schemas && webpack-dev-server --config webpack/webpack.dev.js",
-        "test": "npm run build:webpack:dev && jest -c jest.unit.config.js",
-        "release": "npm run build:webpack:prod && npm run build:es && npm run build:cjs",
+        "server": "webpack-dev-server --config webpack/webpack.dev.js",
```

Comment:
Is `build:schemas` triggered here, or is it intentionally removed?

Reply:
Intentionally removed